crates/burn-import/src/onnx/dim_inference.rs

```diff
@@ -266,6 +267,33 @@ fn reduce_mean_update_outputs(node: &mut Node) {
     }
 }
 
+/// Update the output tensor dimension
+fn squeeze_update_output(node: &mut Node) {
+    if let Some(Data::Int64s(axes)) = &node.inputs[1].value {
+        if axes.len() != 1 {
+            panic!("Squeeze: Only one axis should be specified for squeezing.");
```

Comment:
@antimora do we want full squeeze support? Multiple axes can be specified according to [the spec](https://github.com/onnx/onnx/blob/main/docs/Changelog.md#Squeeze-13), which would translate to multiple `.squeeze(dim)` calls in Burn code.

Reply:
Good point. It skipped me too.

Reply:
The `unsqueeze` op has `unsqueeze`, `unsqueeze_dim`, and `unsqueeze_dims` in burn. If we want to follow the same strategy, we could add a `squeeze_dims` op in burn, although that would probably belong in another PR as @laggui said.

Reply:
Sounds good. We can start with partial support (only one squeeze axis) with this PR.